bindings/c/test/apitester/TesterBlobGranuleCorrectnessWorkload.cpp

```diff
@@ -218,6 +227,62 @@ class ApiBlobGranuleCorrectnessWorkload : public ApiWorkload {
 		    getTenant(tenantId));
 	}
 
+	void validateRanges(std::shared_ptr<std::vector<fdb::KeyRange>> results,
```

Comment:
This refactoring should be shared with randomGetRangesOp

Reply:
Oh yes. You are right. I have pushed another commit to include your entire refactoring.